mysql-test/suite/innodb/t/parallel_dblwr_encrypt.test

```diff
@@ -0,0 +1,92 @@
+# Test Parallel double write buffer encryption
+
+--source include/have_innodb.inc
+--source include/not_embedded.inc
+--source include/have_debug.inc
```

Comment:
This does not have to be a debug testcase: issue workload, wait until buffer pool is clean, kill/shutdown mysqld - the doublewrite will have lots of page images from previous flushes for grepping

Reply:
Done